src/Plugin.php

```diff
@@ -26,6 +26,14 @@ class Plugin {
    */
   const L10N = self::PREFIX;
 
+  /**
+   * Cron event name for checking database indices.
+   *
+   * @var string
+   */
+  const CRON_EVENT_ENSURE_INDICES = Plugin::PREFIX . '/cron/ensure-indices';
```

Comment:
```suggestion
  const CRON_EVENT_ENSURE_INDEXES = Plugin::PREFIX . '/cron/ensure-indexes';
```
Both indices and indexes are valid plural forms, but I'd recommend to use indexes, because developers will probably try to search the code for "index".